### PR TITLE
added information about mock calls CallThrough

### PR DESCRIPTION
**Pull request by gsabatie**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -267,6 +267,11 @@ const spy = jest.spyOn(ajax, 'request').mockImplementation(() => Promise.resolve
 expect(spy).toHaveBeenCalled()
 spy.mockRestore()
 ```
+### Mock object methods and call through to the real function
+
+```js 
+const spyWithCallToRealFunction = jest.spyOn(ajax, 'request').andCallThrough()
```
Comment (lirantal) [inline Readme.md]: Maybe add a comment that this is also the default behavior without adding `andCallThrough()` ?

File: Readme.md [hunk 2]
```diff
@@ -267,6 +267,11 @@ const spy = jest.spyOn(ajax, 'request').mockImplementation(() => Promise.resolve
 expect(spy).toHaveBeenCalled()
 spy.mockRestore()
 ```
+### Mock object methods and call through to the real function
```
Comment (sapegin) [inline Readme.md:270]: I'd add an explanation what “call through” means.

File: Readme.md [hunk 3]
```diff
@@ -288,6 +293,23 @@ Object.defineProperty(location, 'title', {
 })
 ```
 
+### Verify mock calling
```
Comment (sapegin) [inline Readme.md]: I think we have enough examples in the “Matchers” section. It probably could be improved, but adding a new sections that is almost the same isn't a good solution.

**Review by sapegin — CHANGES_REQUESTED** [review]
Thanks! A few notes.

File: Readme.md (hunk 3 above)
Comment (gsabatie) [reply Readme.md]: yes sur

**Pull request closed without merging**